photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -160,6 +171,28 @@ final class Driver(val sc: SparkContext, val params: GameParams, implicit val lo
     }
 
   /**
+   * Run various checks on data (training or validation. Throws a RuntimeException if any of the checks does not pass.
+   *
+   * @note this is somewhat expensive, so we can turn it off
+   *
+   * Checks performed
+   * ================
+   * - all weights >= 0.0 and at least 1 weight > 0
+   *
+   * @param data The data to check
+   */
+  protected[training] def checkData(data: DataFrame): DataFrame =
+
+    if (params.checkData)
```

Comment:
Could you please add some brackets around the body of this `if` - I'd prefer we do that just to be clear and safe (if we're not using the `if` as a ternary operator)